test/integration/client_test.go

```diff
@@ -50,7 +50,6 @@ func TestClient(t *testing.T) {
 	defer s.Close()
 
 	m = master.New(&master.Config{
```

Comment:
See the client_test.go change in #5085 to enable v1beta3 testing.
cc @smarterclayton